### DISCUSSION TITLE
Custom Metrics (remove unneeded polling)

### DISCUSSION THREAD

**Discussion by stephack** [timmo001/system-bridge#1032] [Q&A]
Are there plans to allow users to customize what metrics are polled (to minimize resources and load on the host machine)?
If not, please consider this a feature request (for Windows at least).
Every time a poll is done to update the metrics there is a 5-10% jump in CPU usage. I do not need all the included metrics (File system for example) and it would be great if I could pick and choose what is polled.

Regardless, thanks for the great work done so far.

**Comment by VNRARA** [comment]
This is very needed indeed as I noticed that system bridge has the second largest footprint on my system after my DAW running a project
